app/assets/javascripts/comments.js

```diff
@@ -1,5 +1,29 @@
 $( function () {
-  $('.edit_comment #comment_content').live('change', RescueRails.saveParentForm);
+
+  if ( $('.edit_comment').length > 0) {
+
+    $('.toggle-edit-comment').click(function(){
```

Comment:
k got it (these are the things you don't worry about and forget once you're doing ember =\)

```javascript
$(document).on('click', '.toggle-edit-commit', function(){
```